scripts/test-jvm

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+
+set -eu
+
+clojure -Srepro -A:test:runner:cljs-jvm -r '^cljs-tooling.*' $@
```

Comment:
I'm a bit puzzled that some scripts use lein, some tools.deps. Some explanations would be useful.

Reply:
I found that `lein test` was not working on my machine, it seemed to do some caching. I had `clojure` scripts handy and decided to use those. It will also nice down the road to use this project as git dep.

Reply:
tried again and I get

```
Exception in thread "main" java.lang.RuntimeException: No such var: u/add-ns-macros, compiling:(cljs_tooling/test_util.cljc:7:15)
...
```

weird..I don't know what is going on here...

Reply:
While I agree with you for consistency, for CI the new cli tool is a better tool - faster and resolves deps more precisely. Will try to debug but I cannot promise I can spend a lot of time on this...all tests are green :))))